site/content/resources/security-checklist.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Security Response Checklist
+layout: page
+---
+
+This document outlines a checklist for Contour security team members (at time of writing, this is the same people as the maintainer team) to step through in the event Contour has a CVE that needs to be mitigated.
+
+## A CVE has been reported, what do I do?
+
+1. User discovers a vulnerability and notifies cncf-contour-maintainers@lists.cncf.io
+1. Contour maintainer team triages the vulnerability with the reporter and decide patch releases (multiple minors could be impacted) as well as downstream distributors.
+1. Create a Security Advisory Draft on github Contour repo https://github.com/projectcontour/contour/security/advisories
+    - Requires patched versions 
+    - As part of this, fill out the CVSS score and CWE enumerator, and request a CVE ID via Github.
+1. Do not publish draft, keeping it in draft mode until we release patch
+    - Remember to give credit to the reporter, they can however remain anonymous or keep their company info private if they wish
+1. Communicate to the reporter that draft is created & awaiting for precise dates for releases
+1. Send email to the Distributors (cncf-contour-distributors-announce@lists.cncf.io) mailing list on disclosure and patch releases dates, can include
+    - Learn from previous mistakes, send this through the web interface at https://lists.cncf.io/g/cncf-contour-distributors-announce/ !
+      Don't use a client that may "correct" the address to another one for you.
+    - Description of vulnerability
+    - Contour versions affected
+    - Known attack vectors
+    - Possible workarounds
+    - Next step including patch releases
+    - Leave out the CVE ID
+    - get buy-in from the distributors on release date, or at least see if there are objections
+    - Post the Embargo note (from https://github.com/projectcontour/contour/blob/main/SECURITY.md)  at the bottom
```

Comment:
The referenced markdown file got removed. So maybe this hint isn't needed?

Reply:
I changed it to a symlink to the new site file, so it's still there. I could change this to reference the site instead, I guess.